Econ490_PS1.do

```diff
@@ -0,0 +1,141 @@
+*Week 4 Assignment
+*Oghenefegor Omorojor
+
+cd "/Users/user/Box Sync/Week4/04_assignment/data"
+
+global wd  "/Users/user/Box Sync"
+
+use "$wd/Week4/04_assignment/data/village_pixel", clear
+
+summarize
+
+describe
+
+*Question 1a
+sort payout pixel
+by payout (pixel), sort: gen pixel_consistent = payout[1] != payout[_N]
+
+
+
+*Question 1b
+* convert pixel variable to categorical variable + inspect categories
+encode pixel, generate(pixel_n)
+tab pixel_n, nolabel
+* creating the dummy
+bysort village: egen mode_pixel = mode(pixel_n), maxmode
+gen pixel_household = 0
+replace pixel_household = 1 if mode_pixel != pixel_n
+bysort village: egen pixel_village = mean(pixel_household)
+replace pixel_village = 1 if pixel_village > 0
+tab pixel_village
+* Identifying village names
+gen household1 if mode_pixel = 1
+
+
+*Question 1c
+gen household2 if mode_pixel != 1 | pixel_consistent = 0
+gen household3 if mode_pixel != 1 | pixel_consistent = 1
+
+
+*Question 2
```

Comment:
The code is not working. You might consider using the psle_student_raw.dta directly and removing the data extraction part. Additionally, you need to create a loop that parses each school's information individually.

Reply:
A redo has been committed